_episodes/01-introduction.md

```diff
@@ -6,18 +6,21 @@ questions:
 - "What are containers, and why might they be useful to me?"
 objectives:
 - "Show how software depending on other software leads to configuration management problems."
-- "Identify the problems that software installation problems can pose for research."
-- "Give two examples of how containers can solve software configuration problems." 
+- "Identify the problems that software installation can pose for research."
+- "Explain the advantages of application virtualization."
```

Comment:
Looking at this again, I'm not sure if application virtualization is what we're covering here...maybe it should be application containerization? Or maybe this point can be left out altogether?

Reply:
```suggestion
- "Explain the advantages of containerization."
```